test/collection-browser.test.ts

```diff
@@ -5,10 +5,28 @@ import sinon from 'sinon';
 import type { InfiniteScroller } from '@internetarchive/infinite-scroller';
 import type { CollectionBrowser } from '../src/collection-browser';
 import '../src/collection-browser';
+import { defaultSelectedFacets, SortField } from '../src/models';
 import { MockSearchService } from './mocks/mock-search-service';
 import { MockCollectionNameCache } from './mocks/mock-collection-name-cache';
 
 describe('Collection Browser', () => {
+  it('clear existing filter for facets & sort-bar', async () => {
```

Comment:
I'm not clear how this change relates to the plurization ticket?

Reply:
Trying to imporve code coverage for other parts